src/macros.js

```diff
@@ -215,6 +215,7 @@ const def = (context, global: boolean) => {
             throw new ParseError(`Invalid argument number "${arg[0].text}"`);
         }
         numArgs++;
+        // eslint-disable-next-line transform-runtime-aliasing/no-transform-runtime-aliasing
```

Comment:
Adding this wherever we have a `parseInt` or `parseFloat` is kind of a pain.  Do we still need this lint rule now we're using `browserlist`?

Reply:
@kevinbarabash I've added an `ignore` option to `no-transform-runtime-aliasing` and ignored `JSON.stringify`, `parseInt`, and `parseFloat`. which clearly don't need polyfill.